### ISSUE TITLE
Customize error messages

### ISSUE THREAD

**Issue by amyschools** [Tivix/django-rest-auth#349]
Is there currently a way to override the hard-coded error messages in the [serializers.py](https://github.com/Tivix/django-rest-auth/blob/master/rest_auth/serializers.py#L30) file? 

Some options are:

1) Subclassing the LoginSerializer in our code.

2) Submit a PR to allow for customizable messages.

What are your thoughts?

**Comment by snewcomer** [comment]
Yeah also looking at this.  Any thoughts on if we can help with a PR?

**Comment by mariodev** [comment]
@amyschools @snewcomer Thanks for the suggestions. Added PR. With this, you should be able to override specific error message inside `__init__` method. Please lmk if it looks good or needs more work. Thanks again.
